Patch tests to support correct warning messages

diff --git a/tests/test_annotated.py b/tests/test_annotated.py
index 541f84bc..f1a50788 100644
--- a/tests/test_annotated.py
+++ b/tests/test_annotated.py
@@ -57,7 +57,7 @@ foo_is_short = {
             {
                 "ctx": {"min_length": 1},
                 "loc": ["query", "foo"],
-                "msg": "String should have at least 1 characters",
+                "msg": "String should have at least 1 character",
                 "type": "string_too_short",
                 "input": "",
                 "url": match_pydantic_error_url("string_too_short"),
@@ -68,7 +68,7 @@ foo_is_short = {
             {
                 "ctx": {"limit_value": 1},
                 "loc": ["query", "foo"],
-                "msg": "ensure this value has at least 1 characters",
+                "msg": "ensure this value has at least 1 character",
                 "type": "value_error.any_str.min_length",
             }
         )
